db/models.py

```diff
@@ -1 +1,11 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+
+class Actor(models.Model):
+
```

Comment:
a bit inconsistent styling in comparison to the previous model, you should not use an empty line after the class signature, just as you did with the previous model